easybuild/easyconfigs/x/xxd/xxd-8.2.4220-GCCcore-10.2.0.eb

```
@@ -0,0 +1,37 @@
+# Last contribution from the NIHR Biomedical Research Centre
+# Guy's and St Thomas' NHS Foundation Trust and King's College London
+# uploaded by J. Sassmannshausen
+
+easyblock = 'MakeCp'
+
+name = 'xxd'
+version = '8.2.4220'
+
+homepage = 'https://www.vim.org'
+description = """xxd is part of the VIM package and this will only install xxd, not vim!
+xxd converts to/from hexdumps of binary files."""
```

The description is used in the module file, so it should be relevant to the final user. I would remove any mentions to what will be installed
```suggestion
description = "xxd is part of the VIM package and serves to convert to/from hexdumps of binary files."
```